diff --git a/src/content/docs/createFormControl.mdx b/src/content/docs/createFormControl.mdx
index 53d2d317..d807a1ab 100644
--- a/src/content/docs/createFormControl.mdx
+++ b/src/content/docs/createFormControl.mdx
@@ -38,6 +38,8 @@ This function creates the entire form state subscription and allows you to subsc
  ```tsx
  const props = createFormControl()
 
+{" "}
+
 <FormProvider {...props} /> // ❌ You don't need provider
 
  <input {...props.register('name')} /> // ✅ Direct use method from createFormControl
